protos/feast/core/Store.proto

```diff
@@ -147,6 +147,7 @@ message Store {
     string database = 7;
     string schema = 8;
     string warehouse = 9;
+    string tablename = 10;
```

Comment:
I'm using `table` as the key of the config in the retriever, at [pos1](https://github.com/Cimpress-MCP/feast/blob/11550ccbf989e4da74897df37cffd41fa49afb79/storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/SnowflakeQueryTemplater.java#L41), and at [pos2](https://github.com/Cimpress-MCP/feast/blob/11550ccbf989e4da74897df37cffd41fa49afb79/storage/connectors/jdbc/src/test/java/feast/storage/connectors/jdbc/retriever/SnowflakeHistoricalRetrieverJSONColTest.java#L62). So maybe you can update them to `tableName`, when you do rebase ~